benchmarkplaces/providers.py

```diff
@@ -163,6 +163,23 @@ def search_places(self, name, address, **kwargs):
         else:
             raise APIError('An error occurred with %s API' % self.name)
 
+    # yelp place detail request
+    def get_yelp_details(self, yelp_id, **kwargs):
+        url = 'http://api.yelp.com/v2/business/' + yelp_id
```

Comment:
Ohhh okay, that makes more sense because of python weirdness i've noticed, so i'll do it that way from now on.